### DISCUSSION TITLE
How many Plants should I make

### DISCUSSION THREAD

**Discussion by SNora00** [pfazekas1/stellar_dew#12] [Q&A] [answered]
HI! Currently I made 2 plants, will be expanded in the future but could you guys tell me approximately to how many should I make?  And if you have some Ideas for plant names, please share it with me.
Thanks!

**Comment by pfazekas1** [comment, marked as answer]
I think 3 should be enough for now. The classes for the plants are pretty much the same so if we decide to add some more then its easy to implement. Also I think we should keep thing simple for now and if we have time then we can expand the game with more plants.

**Reply by SNora00** [reply]
Okay, thank you! And if you guys have some cool name for the plants, don't keep it in yourself ^^.

**Comment by TimiBodi** [comment]
Cool names? I have a couple but I am not sure if they will be cool enough xd. I have four so far:  Calendula, Vallaria, Nymfaea and Pelargonia. My biologist roommate helped me come up with them and I quite like them. I hope it helped.
